engines/director/debugger/dt-cast.cpp

```diff
@@ -186,13 +188,78 @@ void showCast() {
 						ImGui::Text("%s", toString(castMember._value->_type));
 
 						ImGui::TableNextColumn();
-						ImGuiImage imgID = getImageID(castMember._value);
-						if (imgID.id) {
-							showImage(imgID, name.c_str(), 32.f);
+						float columnWidth = ImGui::GetColumnWidth();
+
+						switch (castMember._value->_type) {
+						case kCastBitmap:
+							{
+								float offsetX = (columnWidth - 32.f) * 0.5f;
+								ImGui::SetCursorPosX(ImGui::GetCursorPosX() + offsetX);
+
+								ImGuiImage imgID = getImageID(castMember._value);
+								if (imgID.id) {
+									showImage(imgID, name.c_str(), 32.f);
+								}
+							}
+							break;
+
+						case kCastText:
+							{
+								const char *text = Common::toPrintable(((TextCastMember *)castMember._value)->_rtext).c_str();
+								ImVec2 textSize = ImGui::CalcTextSize(text);
+								float offsetX = (columnWidth - textSize.x) * 0.5f;
+
+								ImGui::SetCursorPosX(ImGui::GetCursorPosX() + offsetX);
+								ImGui::Text("%s", text);
```

Comment:
While text as is is okay, still it does not serve the right purpose. Properly, you should take the rendered text from MacText and display it as image.